eng/Build.props

```diff
@@ -167,6 +167,13 @@
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Configuration.Configurationmanager.7.0.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Security.Cryptography.Xml.7.0.1.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Threading.Channels.6.0.0.csproj" />
+
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Win32.Registry.5.0.0.csproj" />
```

Comment:
This is an existing package.  Entry is not necessary here.

Reply:
I see...are the packages not in this list built first?

Reply:
Only new package that aren't in previously source-built packages are in this list.  After preview 4 is release and we update the artifacts this list will get cleared out.